docs/github_redaction_failure.md

```diff
@@ -0,0 +1,272 @@
+# How to recover from an opensafely redaction failure
+## output-publisher repos
+
+If you inadvertently commit things that should be redacted to the output-publisher repo, but do not run `osrelease` (and have not done anything out of the ordinary) then you can safely amend your files, commit the results and carry on working. This guide is mostly aimed at study repos.
+
+## make it safe
+* lock down permissions
+* make public repos -> private
+* temporarily remove external collabs
+* check for forks (should be prevented, but best to verify)
```

Comment:
I just disabled this in opensafely org, FYI.

Reply:
great!